docs/standard/modern-web-apps-azure-architecture/index.md

```diff
@@ -20,7 +20,7 @@ One Microsoft Way
 
 Redmond, Washington 98052-6399
 
-Copyright © 2018 by Microsoft Corporation
+Copyright © 2019 by Microsoft Corporation
 
 All rights reserved. No part of the contents of this book may be reproduced or transmitted in any form or by any means without the written permission of the publisher.
 
```

Comment:
not related to your changes but might be related to the markdig migration

Your name is not being bolded as you probably intended on line 41
![image](https://user-images.githubusercontent.com/12971179/52246314-345b8b00-289a-11e9-8d13-165fefa303d5.png)